src/Tools/MicroBuild/run-pdbgit.ps1

```diff
@@ -0,0 +1,70 @@
+<#
+
+    Run PDB Git on our PDB files to enable proper source server support
+
+#>
+Param(
+    [string]$sourcePath = $null,
+    [string]$binariesPath = $null
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+function Get-PackagesPath {
+    $packagesPath = $env:NUGET_PACKAGES
+    if ($packagesPath -eq $null) {
+        $packagesPath = join-path $env:UserProfile ".nuget\packages\"
+    }
+
+    return $packagesPath
+}
+
+try
+{
+    [xml]$deps = get-content (join-path $sourcePath "build\Targets\Dependencies.props")
+    $pdbGitVersion = $deps.Project.PropertyGroup.PdbGitVersion
+    $packagesPath = Get-PackagesPath
+    $pdbGit = join-path $packagesPath "pdbgit\$pdbGitVersion\tools\PdbGit.exe"
+
+    $signToolDataPath = join-path $sourcePath "build\config\SignToolData.json"
```

Comment:
Could a comment be added to explain why we're involving signing configuration here? I have guesses but I'd rather not be guessing and just know.